model-server/src/main/kotlin/org/modelix/model/server/handlers/ui/HistoryHandler.kt

```diff
@@ -313,6 +313,17 @@ class HistoryHandler(val client: IModelClient, private val repositoriesManager:
                     }
                 }
             }
+
+            val previousVersion = version.baseVersion
+            if (previousVersion != null) {
```

Comment:
nit: The buttons are not aligned for the inital version.

![image](https://github.com/user-attachments/assets/e882613f-813e-41ac-ba61-d9fc0d7abfc9)

Reply:
fixed